tardis/model/tests/data/csvy_model_to_test_abundances.csvy

```diff
@@ -12,9 +12,6 @@ datatype:
     -  name: density
        unit: g/cm^3
        desc: density of shell.
-    -  name: t_rad
-       unit: K
-       desc: radiative temperature.
```

Comment:
I know we don't use t_rad for anything in this test, so in that sense it's unnecessary, but maybe it's better to only make more important changes in the PR?  @harpolea what do you think?

Reply:
I think it's good to have tests as clean as possible so that they don't contain anything unnecessary (and therefore contain as little that can go wrong as possible), so I think this is ok